<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkWindow</class>
 <widget class="QWidget" name="networkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>network remote</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="3">
    <widget class="QSpinBox" name="spb_ticks">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLineEdit" name="txt_password">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="3">
    <widget class="QSpinBox" name="spb_reset">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>3600</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLabel" name="lbl_status">
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>pausing</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="4">
    <widget class="QLabel" name="lbl_ticksToNext">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QCheckBox" name="chb_reset">
     <property name="text">
      <string>reset every</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_password_lbl">
     <property name="text">
      <string>password:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lbl_wait_lbl">
     <property name="text">
      <string>wait</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_status_lbl">
     <property name="text">
      <string>status:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="btn_info">
     <property name="text">
      <string>info</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_username_lbl">
     <property name="text">
      <string>username:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_url_lbl">
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbl_every_lbl">
     <property name="text">
      <string>every</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="lbl_ticksToNext_lbl">
     <property name="text">
      <string>ticks to next action:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spb_ticksWait">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="4">
    <widget class="QLabel" name="lbl_waitToScreenUpload">
     <property name="text">
      <string>ticks to screen upload</string>
     </property>
    </widget>
   </item>
   <item row="10" column="6">
    <widget class="QLabel" name="lbl_resetTicks_lbl">
     <property name="text">
      <string>ticks</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="6">
    <widget class="QLineEdit" name="txt_username">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QLineEdit" name="txt_url">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>ftp://example.com/mlab</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLabel" name="lbl_ticks">
     <property name="text">
      <string>ticks</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="btn_startStop">
     <property name="text">
      <string>start</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="chb_downloadCmds">
     <property name="text">
      <string>download cmds</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="3">
    <widget class="QCheckBox" name="chb_uploadScreen">
     <property name="text">
      <string>upload screen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="7">
    <widget class="QLabel" name="lbl_info">
     <property name="text">
      <string>Specify parameter...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spb_ticks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
